app/Repositories/WorkingScheduleRepository.php

```diff
@@ -11,11 +13,62 @@ public function model()
 
     public function getUserSchedule($id)
     {
+        $this->makeModel();
+
         return $this->model->where('user_id', $id)->pluck('shift', 'date');
     }
     
     public function getLocation($id)
     {
+        $this->makeModel();
+
         return $this->model->where('user_id', $id)->pluck('location_id', 'date');
     }
+
+    public function getByMonth($request)
+    {
+        $this->makeModel();
+
+        $users = User::with([
+            'workspace',
+            'position',
+            'program',
+        ])
+        ->where(function ($query) use ($request) {
+            if ($request->has('workspace') && $request->workspace !== null) {
+                $query->where('workspace_id', $request->workspace);
+            }
+            if ($request->has('program') && $request->program !== null) {
+                $query->where('program_id', $request->program);
+            }
+            if ($request->has('position') && $request->position !== null) {
+                $query->where('position_id', $request->position);
+            }
+        })
+        ->get();
+
+        $first_day = Date(Date('Y') . '-' . Date('m') . '-' . '01');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/WorkingScheduleRepository.php</br>**Line**: 50</br>Variable "first_day" is not in valid camel caps format